kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/ExpressionUtils.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (2023) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.utils
+
+import io.delta.kernel.expressions.{Column, Expression, Literal, Predicate}
+
+/** Useful helper functions for creating expressions in tests */
```

Comment:
we already have one here with the same name: io.delta.kernel.defaults.internal.expressions.ExpressionUtils.java

Reply:
Also I think this is better move into the API module (internal only)

Reply:
+1 consolidate please.

Reply:
I've created `ExpressionUtils` in the kernel-api module and had already renamed this to `ExpressionTestUtils`. Not sure if we want to convolute that with all these methods--especially since I've included some unsupported expressions?